src/main/java/com/example/recipes/model/Recipe.java

```diff
@@ -0,0 +1,69 @@
+package com.example.recipes.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+@EqualsAndHashCode
+@ToString
+public class Recipe {
+    private String recipeName;
+    private int timeForPreparing;
+
+    private Ingredient ingredient;
+    private List<String> cookingSteps = new LinkedList<>();
+
+    private int id = 0;
+
+    public Recipe(String recipeName, int timeForPreparing, Ingredient ingredient, List<String>cookingSteps, int id) {
+        this.recipeName = recipeName;
+        setTimeForPreparing(timeForPreparing);
+        this.ingredient = ingredient;
+        this.cookingSteps.add(String.valueOf(cookingSteps));
+        this.id = id;
+    }
+
```

Comment:
**public Recipe(String recipeName, int timeForPreparing, Set<Ingredient> ingredients, List<String>cookingSteps) {
        this.recipeName = recipeName;
        setTimeForPreparing(timeForPreparing);
        this.ingredient = ingredients;
        this.cookingSteps.add(String.valueOf(cookingSteps));
        this.id = counter++;
}**